mhkit/dolfyn/io/nortek_defs.py

```diff
@@ -340,3 +340,144 @@ def sci_func(self, data):
+                         default_val=nan,
+                         factor=0.1,
+                         units='degree',
+                         long_name='Roll',
+                         standard_name='platform_roll',
+                         ),
+    'pressure1_alt': _VarAtts(dims=[],
+                              dtype=np.float32,
+                              group='data_vars',
+                              default_val=nan,
+                              factor=0.001,
+                              units='dbar',
+                              long_name='Pressure Min',
+                              standard_name='sea_water_pressure',
+                              ),
+    'pressure2_alt': _VarAtts(dims=[],
+                              dtype=np.float32,
+                              group='data_vars',
+                              default_val=nan,
+                              factor=0.001,
+                              units='dbar',
+                              long_name='Pressure Max',
+                              standard_name='sea_water_pressure',
+                              ),
+    'temp_alt': _VarAtts(dims=[],
+                         dtype=np.float32,
+                         group='data_vars',
+                         default_val=nan,
+                         factor=0.01,
+                         units='degree_C',
+                         long_name='Temperature',
+                         standard_name='sea_water_temperature',
+                         ),
+}
+
+waves_data = {
+    'pressure_alt': _VarAtts(dims=[],
+                             dtype=np.float32,
+                             group='data_vars',
+                             default_val=nan,
+                             factor=0.001,
+                             units='dbar',
+                             long_name='Pressure',
+                             standard_name='sea_water_pressure',
+                             ),
+    'dist1_alt': _VarAtts(dims=[],
+                          dtype=np.uint16,
+                          group='data_vars',
+                          default_val=nan,
+                          factor=0.001,
+                          units='m',
+                          long_name='AST distance1 on vertical beam',
+                          standard_name='altimeter_range',
+                          ),
+    'dist2_alt': _VarAtts(dims=[],
+                          dtype=np.uint16,
+                          group='data_vars',
+                          default_val=nan,
+                          factor=0.001,
+                          units='m',
+                          long_name='AST distance2 on vertical beam',
+                          standard_name='altimeter_range',
+                          ),
+    'AnaIn1_alt': _VarAtts(dims=[],
+                           dtype=np.float32,
+                           group='sys',
+                           default_val=nan,
+                           units='n/a',
```

Comment:
Should this one have no name keys?

Reply:
This variable isn't used, so I don't add them. This is the raw ADC output, I believe